Automattic/wp-calypso
# Sites Management Page: Update status and sorting dropdown labels

#### Proposed Changes

In order to improve accessibility, let's add labels so it's clear for screen readers that you're touching a dropdown with a value instead of just a button.

![image](https://user-images.githubusercontent.com/26530524/190715133-bb361f5f-4768-4275-b552-db518411cb74.png)

#### Testing Instructions

Enable the screen reader and tab through the dropdowns. The spoken message should match the intent of the button, which should indicate that the user is touching a select box.
